compute-a-16-49.o2.rc.hms.harvard.edu
removed gaps!	 filt_gaps: 0.00095367431640625 seconds
num cpus: 4
[612. 408. 204.   0.]
num_jobs:  4
[816. 612. 408. 204.]
------------------------new process------------
start,end:612.0,817.0
processes initialized!
process appended!
------------------------new process------------
start,end:408.0,612.0
processes initialized!
process appended!
------------------------new process------------
start,end:204.0,408.0
processes initialized!
process appended!
------------------------new process------------
start,end:0.0,204.0
processes initialized!
process appended!
4
process started! <Process(Process-1, started)> True
process started! <Process(Process-2, started)> True
process started! <Process(Process-3, started)> True
Process Process-2:
Traceback (most recent call last):
  File "/n/app/python/3.6.0/lib/python3.6/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/n/app/python/3.6.0/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "multithreading.py", line 52, in f
    for i in range(start_ind,end_ind):
TypeError: 'numpy.float64' object cannot be interpreted as an integer
Process Process-4:
Traceback (most recent call last):
  File "/n/app/python/3.6.0/lib/python3.6/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/n/app/python/3.6.0/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "multithreading.py", line 52, in f
    for i in range(start_ind,end_ind):
TypeError: 'numpy.float64' object cannot be interpreted as an integer
Process Process-1:
Traceback (most recent call last):
  File "/n/app/python/3.6.0/lib/python3.6/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/n/app/python/3.6.0/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "multithreading.py", line 52, in f
    for i in range(start_ind,end_ind):
TypeError: 'numpy.float64' object cannot be interpreted as an integer
Process Process-3:
Traceback (most recent call last):
  File "/n/app/python/3.6.0/lib/python3.6/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/n/app/python/3.6.0/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "multithreading.py", line 52, in f
    for i in range(start_ind,end_ind):
TypeError: 'numpy.float64' object cannot be interpreted as an integer
slurmstepd: error: *** JOB 48433620 ON compute-a-16-49 CANCELLED AT 2019-08-14T16:20:18 ***
